README.md

```diff
@@ -51,6 +65,20 @@ export default YourPage
 | style | object | `{}` | Directly set the style object of the svg container |
 | lineStyle | object | `{}` | Set the style object of the line |
 
+### Donut
+
+| Property | Type | Default | Description |
+| - | - | - | - |
+| data | array | `[]` | An array of numbers |
+| width | string | - | CSS width of your chart |
+| height | string | `100%` | CSS height of your chart |
+| colors | string | `CATEGORICAL` | CSS colours used for each data point |
+| style | object | `{}` | Directly set the style object of the svg container |
+| offset | number | 0 | Starting point on circle for the first segment as a portion of a full rotation |
+| hole | number | 0.5 | Portion of the diameter used for each segments inner radius |
```

Comment:
Maybe this could be "Percentage of the diameter that is cut out from the center"